src/util/map.js

```diff
@@ -65,7 +65,8 @@ export const toGeoJson = organisationUnits =>
         .filter(
             ({ geometry }) =>
                 Array.isArray(geometry.coordinates) &&
-                geometry.coordinates.length
+                geometry.coordinates.length &&
```

Comment:
Are both needed or can you just have `geometry.coordinates.flat().length`

Reply:
The argument for keeping both is that flat() can be an expensive operation when there are thousands of coordinates.